File: app/src/main/java/com/shen/xi/android/tut/weibo/WeiboTimeline.java
package com.shen.xi.android.tut.weibo;

import com.google.api.client.util.Key;

import java.util.List;

public class WeiboTimeline {
    @Key("total_number")
    public int totalNumber;
    @Key
    public List<WeiboStatus> statuses;
}

File: app/src/main/scala/com/shen/xi/android/tut/weibo/WeiboTimeline.scala
package com.shen.xi.android.tut.weibo

import com.google.api.client.util.Key
import java.util

final class WeiboTimeline {
  @Key("total_number")
  var totalNumber: Int = -1
  @Key
  var statuses: util.List[WeiboStatus] = null
}


